scripts/net-stats.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+#
+# Reports network statistics
+#
+
+cd "$(dirname "$0")"
+
+# shellcheck source=scripts/configure-metrics.sh
+source configure-metrics.sh
+
+[[ $(uname) == Linux ]] || exit 0
```

Comment:
@pgarg66 and I talked in IRL about sending the diff since the last netstat run rather than the abs value